nodejs/week5/src/frontend/pages/meal.js

```diff
@@ -0,0 +1,150 @@
+      if (res.length === 0) {
+        fetch(`/api/meals/${params.id}`)
+          .then((res) => res.json())
+          .then(
+            (res) =>
+              (document.body.innerHTML = `
+                    <header class="meal">
+                        <h1>Meals Information</h1>
+                        <div>
+                            <a href="/" data-navigo>Home</a>
+                            <a href="/meals" data-navigo>Meals</a> 
+                        </div>
+                    </header>
+            
+                    <main class="meal">
+                        <h3>Meal name:   ${res[0].title}</h2>
+                        <p>Description: ${res[0].description}</p>
+                        <p>Price :${res[0].price}</p>
+                        <footer class="meal">
+                        <p>Telephone</p>
+                        <p>Adress</p>
+                    </footer>`)
+          );
+      } else {
+        document.body.innerHTML = `
+                <header class="meal">
+                    <h1>Meals Information</h1>
+                    <div>
+                        <a href="/" data-navigo>Home</a>
+                        <a href="/meals" data-navigo>Meals</a> 
+                    </div>
+                </header>
+                <main class="meal">
+                    <h3>Meal name:   ${res[0].title}</h3>
+                    <p>Description: ${res[0].description}</p>
+                    <p>Price : ${res[0].price} kr</p>
+                    <form class="reservation" id="form">
+                    <h3>Reserve for this meal</h3>
+                    <div class="reserveInput">
+                        <input type="number" class="guests" id="guests" name="number_of_guests" placeholder="Number of guests">
+                    </div>
+                    <div class="reserveInput">
+                        <input type="datetime" class="created_date" name="for_when" id="created_at" placeholder="Created date"/>
+                    </div>
+                    <div class="reserveInput">
+                        <input type="text" class="name" id="name" name="name" placeholder="Name">
+                    </div>
+                    <div class="reserveInput">
+                        <input type="email" class="email" name="email" id="email" placeholder="Email">
+                    </div>
+                    <div class="reserveInput">
+                        <input type="text" class="phone" name="phone_number" id="phonenumber" placeholder="Phonenumber">
+                    </div>
+                </form>
+                <div>
+                    <button type="submit" class="bookButton" id="button" onClick="makeNewReservaton(${res[0].id})">Book seat</button>   
+                </div>
+            </main>
+            <footer class="meal">
+                <p>Telephone</p>
+                <p>Adress</p>
+            </footer>`;
+      }
+    });
+};
+
+/*(window.handleMealRequest = (params) => {
```

Comment:
It's best to delete commented code. It only really leads to confusion on the part of the developers. If you want to "keep" it, that is what version control is for--commit it, write down the commit id, and then delete it.